showcase/src/assets/translations/en.json

```diff
@@ -352,6 +352,14 @@
           "TITLE": "Based on Angular Material"
         },
         "TITLE": "Stark Typography"
+      },
+      "LAYOUT": {
+        "TITLE": "Layout",
+        "INTRO": "We suggest using <a href=\"https://github.com/angular/flex-layout#angular-flex-layout\" target=\"_blank\" rel=\"noopener noreferrer\">Angular Flex-Layout</a> as a helper library when constructing the layout of your app. Full documentation is available on their <a href=\"https://github.com/angular/flex-layout/wiki/API-Documentation\"  target=\"_blank\" rel=\"noopener noreferrer\">wiki</a>.",
+        "ANGULAR_FLEX_LAYOUT": "Angular Flex Layout",
+        "FLEX": "Flexbox",
+        "GRID_WARNING": "Warning, although certain grid directives are available through the Angular Flex-Layout library they are not completely documented.",
```

Comment:
As we discussed, we should also warn the developer that it doesn't work properly on Chrome 56 :p

Reply:
I added an additional warning about browser support and commented the code with a FIXME. 😄